### DISCUSSION TITLE
What does the name mean?

### DISCUSSION THREAD

**Discussion by four43** [Kludex/mangum#169] [Q&A] [answered]
Just curious where you got the name "Mangum"? I tried looking and it just seems to be a place in Oaklahoma, USA.

**Comment by jordaneremieff** [comment, marked as answer]
It is the last name of my favorite musician, [Jeff Mangum](https://en.wikipedia.org/wiki/Jeff_Mangum) of Neutral Milk Hotel. :)

**Reply by four43** [reply]
Lol awesome. Love it.
